mu/modes/debugger.py

```diff
@@ -17,11 +17,12 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
 import logging
-import os.path
-from mu.modes.base import BaseMode
-from mu.logic import DEBUGGER_PORT
-from mu.debugger.client import Debugger
-from mu.debugger.utils import is_breakpoint_line
+import os
+
+from .base import BaseMode
+from ..debugger.config import DEBUGGER_PORT
+from ..debugger.client import Debugger
+from ..debugger.utils import is_breakpoint_line
```

Comment:
TBH it's a slight personal preference because it feels slightly neater to me.

The advantage is that there's no risk (however slight) of your picking up from some *other* "mu" package which gets ahead on sys.path of the one you're in. The disadvantage is that you can't run the modules standalone, ie you can't do `python mu\logic.py` or whatever, because that module won't be operating inside a package context.

Reply:
Okay, good points.